### PR TITLE
Add `interface_meta` Python package.

### PR DESCRIPTION
**Pull request by matthewwardrop**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by matthewwardrop** [comment]
@conda-forge/help-python Ready for review.

File: recipes/interface_meta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: 68bcafb9dd659f0043ef4d61f2bf86f84114a49423777bfec98fdf46174e2c00
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - decorator
```
Comment (npavlovikj) [inline recipes/interface_meta/meta.yaml:19]: Are you sure you need all of these dependencies here? Usually, only `pip` and `python` are enough for the `host` section.

Comment (matthewwardrop) [reply recipes/interface_meta/meta.yaml:19]: No :). I largely used the pypi skeleton as a starting place, which included these dependencies. As I think about it, I think you are right. `interface_meta` need not ever be imported on the host, and so these deps are unnecessary. I'll patch it!

**Comment by matthewwardrop** [comment]
@npavlovikj Thanks for the review! Should be good now :).

**Comment by npavlovikj** [comment]
Thank you @matthewwardrop !

**Pull request merged**